modules/components/org.wso2.carbon.appfactory.core/src/main/java/org/wso2/carbon/appfactory/core/workflow/TenantCreationDefaultWorkflowExecutor.java

```diff
@@ -0,0 +1,108 @@
+/*
+ *   Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *   WSO2 Inc. licenses this file to you under the Apache License,
+ *   Version 2.0 (the "License"); you may not use this file except
+ *   in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package org.wso2.carbon.appfactory.core.workflow;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.appfactory.common.AppFactoryException;
+import org.wso2.carbon.appfactory.core.services.AppFactoryTenantInfraStructureInitializerService;
+import org.wso2.carbon.appfactory.core.workflow.dto.TenantCreationWorkflowDTO;
+import org.wso2.carbon.appfactory.core.workflow.dto.WorkflowDTO;
+
+/**
+ * The executor class for the default workflow execution
+ */
+public class TenantCreationDefaultWorkflowExecutor implements WorkflowExecutor {
+
+    private static Log log = LogFactory.getLog(TenantCreationDefaultWorkflowExecutor.class);
```

Comment:
This is fixed in following commit.
https://github.com/wso2/product-af/pull/664/commits/6ddbe7623eeb6fdc99f0e16cadec68bdb1ffe0ea